docs/getting-started.md

```diff
@@ -34,221 +37,357 @@ The high level data flow is as follows:
 
 ## 5 Query your local Graph Node
-With your subgraph deployed to your locally running Graph Node, visit http://127.0.0.1:8000/ to open up a [Graphiql](https://github.com/graphql/graphiql) interface where you can explore your deployed GraphQL API for your subgraph by issuing queries and viewing the schema.
+With your subgraph deployed to your locally running Graph Node, visit http://127.0.0.1:8000/ to open up a [Graphiql](https://github.com/graphql/graphiql) interface where you can explore your deployed GraphQL API for your subgraph by issuing queries and viewing the schema. 
 
-See the [Query API](graphql-api.md#1-queries) for a complete reference on how to query your subgraph's entities.
+We have provided a few simple examples below, but please see the [Query API](graphql-api.md#1-queries) for a complete reference on how to query your subgraph's entities.
 
 #### Example
 Query all `Token` entities:
 ```graphql
-query {
+{
   tokens {
     id
-    owner
+    currentOwner
   }
 }
 ```
+Notice that it is just simply `tokens` pluralized, and it will return all of the entites in The Graph. 
+
+Later, when you've deployed your subgraph with this entity, you'll be able to query for a specific value, such as the token ID
+
+```graphql
+{
+  token(id: "c2dac230ed4ced84ad0ca5dfb3ff8592d59cef7ff2983450113d74a47a12") {
+    currentOwner
+  }
+}
+```
+
+You can also organize the queries. This query below would organize all tokens by their ID, and return the current owner of each token. 
+
+```graphql
+{
+  tokens(orderBy: id) {
+    currentOwner
+  }
+}
+```
+
+GraphQL has a ton of functionality! Once again, check out the [Query API](graphql-api.md#1-queries) to figure out how to use all query functionalities. 
+
+## 6 Changing your schema, mappings, and manifest, and launching a new Subgraph
+
+When you first start building your subgraph, it is likely you will make a few iterations to the manifest, mappings, or schema. If you update any of them, you should run `yarn codegen` and `yarn build-ipfs`. This will post the new files on ipfs and give you a new `subgraph ID`, which your graph node can use to track. Note that your graph node can track multiple subgraphs, so you can do this as many times as you like
+
+With the new `subgraph ID`, you can run the exact same `cargo run -p graph-node` command, and just replace the subgraph ID. It is a good idea to also update <YOUR_SUBGRAPH_NAME> so that you can distingush the difference between the subgraphs. <YOUR_SUBGRAPH_NAME> will be used in the endpoint of the GraphiQL server. 
+
+Also, if you set up `graph deploy` as outlined in the [graph-cli](https://github.com/graphprotocol/graph-cli) repository, your graph node should automatically start tracking any new `subgraph ids` created from changes uploaded from `yarn build-ipfs`. 
+
+## 7 Example Subgraphs You can look to 
+
```

Comment:
This is great! Do we want to add `memefactory` as well?

Reply:
added